src/fiatconnect/slice.ts

```diff
@@ -40,6 +49,11 @@ export interface State {
   selectFiatConnectQuoteLoading: boolean
   sendingFiatAccountStatus: SendingFiatAccountStatus
   kycTryAgainLoading: boolean
+  cachedQuoteParams: {
+    [providerId: string]: {
+      [kycSchema: string]: CachedQuoteParams
```

Comment:
I'm purposely using `[kycSchema: string]` here rather than `[kycSchema in KycSchema]`. By specifying the index key to be a member of the `KycSchema` enum, the root state schema will need to updated whenever KycSchema` changes, which seems like an unintended side effect. Whenever we look through `cachedQuoteParams` to see if there's a match when a user presses the KYC Approved homecard/push notification, we'll already have access to the `kycSchema` in question, so we would never need to _use_ the `kycSchema` key,  which would require us to include a type assertion in order to use it effectively.

Reply:
That seems fine to me, Root state is not kind to schemas that are expected to change often